src/Tickets/Commerce/Flag_Actions/Decrease_Stock.php

```diff
@@ -111,14 +111,6 @@ public function handle( Status_Interface $new_status, $old_status, \WP_Post $pos
 				continue;
 			}
 
-			// Is ticket shared capacity?
```

Comment:
Did we replace this with anything?

Reply:
Nope, the issue that was in place before HF will be there again.